BUILD.gn

```diff
@@ -103,6 +103,11 @@ group("brave") {
       ":generate_version",
       "//chrome",
     ]
+    if (enable_brave_vpn) {
+      if (is_win) {
+        deps += [ "//brave//vpn" ]
```

Comment:
I'm not sure it makes sense to build this here because it needs to be signed to work correctly, right? Probably makes more sense to make this a dep of create_dist and also it should be a data_dep

Reply:
removed